insights/parsers/modinfo.py

```diff
@@ -1,36 +1,12 @@
 """
-ModInfo - Commands ``modinfo <module_name>``
-============================================
-Parsers to parse the output of ``modinfo <module_name>`` commands.
-
-ModInfoI40e - Command ``modinfo i40e``
---------------------------------------
-
-ModInfoVmxnet3 - Command ``modinfo vmxnet3``
---------------------------------------------
-
-ModInfoIgb - Command ``modinfo igb``
-------------------------------------
-
-ModInfoIxgbe - Command ``modinfo ixgbe``
-----------------------------------------
-
-ModInfoVeth - Command ``modinfo veth``
---------------------------------------
-
-ModInfoEach - Command ``modinfo *``
------------------------------------
-for any module listed by ``lsmod``
-
-ModInfoAll - Command ``modinfo *(all modules)``
------------------------------------------------
-for all modules listed by ``lsmod``
-
+ModInfoAll - Command ``modinfo *(filtered modules)``
```

Comment:
Since there is a `All` in the name `ModInfoAll`, it makes me think this is the parser for all the modules loaded on this host. How about using another name? like `ModuleInfo`

Reply:
@xiangce, good to me, I'll update it.